SQA_AT/src/main/java/assertion/BaseAssertion.java

```diff
@@ -0,0 +1,39 @@
+package assertion;
+
+import io.restassured.response.Response;
+import org.assertj.core.api.SoftAssertions;
+
+import java.net.HttpURLConnection;
+
+public class BaseAssertion {
+
+    public static void checkResponse(Response response, int statusCode) {
+        SoftAssertions softAssert = new SoftAssertions();
```

Comment:
We can use soft assertions with lambdas e.g.
|
        SoftAssertions.assertSoftly(softly -> {
            softly.assertThat(response.statusCode())
                    .as("Has status code")
                    .isEqualTo(statusCode);
            softly.assertThat(response.contentType())
                    .as("Has content type")
                    .isEqualTo("application/json");
        });